test/unit/common/oatbox/cache/ItemPoolSimpleCacheAdapterTest.php

```diff
@@ -0,0 +1,192 @@
+        $this->cacheItemMock = $this->createMock(CacheItemInterface::class);
+
+        $this->subject = new ItemPoolSimpleCacheAdapter();
+        $this->subject->setServiceLocator(
+            $this->getServiceLocatorMock(
+                [
+                    SimpleCache::SERVICE_ID => $this->cacheMock,
+                ]
+            )
+        );
+    }
+
+    public function testGetItemThatExistInCache(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('get')
+            ->with('key')
+            ->willReturn('value');
+
+        $result = $this->subject->getItem('key');
+        $this->assertInstanceOf(CacheItem::class, $result);
+        $this->assertSame('value', $result->get());
+        $this->assertSame('key', $result->getKey());
+    }
+
+    public function testGetItemThatNotExistInCache(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('get')
+            ->with('key')
+            ->willReturn(null);
+
+        $result = $this->subject->getItem('key');
+        $this->assertInstanceOf(CacheItem::class, $result);
+        $this->assertSame('key', $result->getKey());
+        $this->assertNull($result->get());
+    }
+
+    public function testGetItems(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('setMultiple')
+            ->with(['key1', 'key2']);
+
+        $this->subject->getItems(['key1', 'key2']);
+    }
+
+    public function testHasItem(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('has')
+            ->with('key');
+
+        $this->subject->hasItem('key');
+    }
+
+    public function testClear(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('clear');
+
+        $this->subject->clear();
+    }
+
+    public function testDeleteItem(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('delete')
+            ->with('key');
+
+        $this->subject->deleteItem('key');
+    }
+
+    public function testDeleteItems(): void
+    {
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('deleteMultiple')
+            ->with(['key1', 'key2']);
+
+        $this->subject->deleteItems(['key1', 'key2']);
+    }
+
+    public function testSave(): void
+    {
+        $this->cacheItemMock
+            ->expects($this->once())
+            ->method('getKey')
+            ->willReturn('key');
+
+        $this->cacheItemMock
+            ->expects($this->once())
+            ->method('get')
+            ->willReturn('value');
+
+        $this->cacheMock
+            ->expects($this->once())
+            ->method('set')
+            ->with('key', 'value')
+            ->willReturn(true);
+
+        $this->subject->save($this->cacheItemMock);
```

Comment:
I think we should asset the responses of the methods, they are just annotations and can be wrong. Wdyt?